DirectXMesh/DirectXMeshOptimize.cpp

```diff
@@ -17,758 +17,8 @@
 
 using namespace DirectX;
 
-#pragma warning(disable : 4351)
-
 namespace
 {
-    //---------------------------------------------------------------------------------
```

Comment:
Moved original Hoppe ``OptimizeFaces`` to new file. This file now just includes attribute sort and the ``OptimizeVertices`` function.